tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2018",
```

Comment:
We build both es5 and newer module versions separately (see https://github.com/arkon/ng-click-outside/blob/master/tsconfig.module.json).

That said, I don't really remember why, but I think it broke some people's builds when I jumped up at some point.